gpu/kinfu/tools/kinfu_app.cpp

```diff
@@ -132,14 +132,14 @@ namespace pcl
           * \return true if the operation was successful (the handler is capable and 
           * the input cloud was given as a valid pointer), false otherwise
           */
```

Comment:
You need to update the docstring. Any idea why this change was required in this PR? I would rather do this in a separate PR. Otherwise it's gonna get buried here.

Reply:
It's currently overriding a deprecated method https://github.com/PointCloudLibrary/pcl/blob/5e8f8beb4f6793b507154b80fa269f02eb51f41c/visualization/include/pcl/visualization/point_cloud_color_handlers.h#L113-L122

But that doesn't seem to spawn any warnings.

I'll move it to a separate PR.